scene/3d/decal.cpp

```diff
@@ -207,7 +230,7 @@ void Decal::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "emission_energy", PROPERTY_HINT_RANGE, "0,128,0.01"), "set_emission_energy", "get_emission_energy");
 	ADD_PROPERTY(PropertyInfo(Variant::COLOR, "modulate"), "set_modulate", "get_modulate");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "albedo_mix", PROPERTY_HINT_RANGE, "0,1,0.01"), "set_albedo_mix", "get_albedo_mix");
-	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "normal_fade", PROPERTY_HINT_RANGE, "0,1,0.01"), "set_normal_fade", "get_normal_fade");
```

Comment:
What is the benefit of using ``0.999`` over ``1``, or ``1.0``?

Reply:
The decal becomes invisible if you set normal fade to 1.0, even if the decal is completely perpendicular to a surface.

Reply:
Makes sense!

For future reference, this is the place where normal_fade is used https://github.com/godotengine/godot/blob/fc00a8390193220cea702d9ab684fb6c53d97a6b/servers/rendering/renderer_rd/shaders/scene_forward_clustered.glsl#L803